recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,4 @@
+CONDA_BUILD_SYSROOT:       # [osx]
+  - /opt/MacOSX10.12.sdk   # [osx]
```

Comment:
Remove these 2 lines

Reply:
We need these for Anaconda Distro, are they harmful?

Reply:
Yes, it's harmful for conda-forge as we are installing the SDK somewhere else.

Reply:
OK, I'm probably OK with aligning our location then. We do need to force 10.12 somehow though. What is the default ATM? Any clues how to force 10.12?

Reply:
It is set to `$(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX${MACOSX_DEPLOYMENT_TARGET}.sdk` which wade sense when we were using system compilers.

Default is 10.9. If you set MACOSX_DEPLOYMENT_TARGET to 10.12 as you've done here in `recipe/conda_build_config.yaml`, the CI script will read that variable, download 10.12, extract it and automatically set `CONDA_BUILD_SYSROOT`. So there's no need to set `CONDA_BUILD_SYSROOT` in the recipe. Just `MACOSX_DEPLOYMENT_TARGET` would do.